astropy/visualization/mpl_style.py

```diff
@@ -26,12 +24,13 @@
 example), you should reset the matplotlib settings to the library defaults
 *before* applying the astropy style, e.g.::
 
-    >>> import matplotlib as mpl
-    >>> from astropy.visualization import astropy_mpl_style
-    >>> mpl.rcdefaults()
-    >>> mpl.rcParams.update(astropy_mpl_style)
+>>> import matplotlib as mpl
+>>> from astropy.visualization import astropy_mpl_style
+>>> mpl.rcdefaults()
+>>> mpl.rcParams.update(astropy_mpl_style)
 """
 
+__doctest_requires__ = {'*': ['matplotlib']}
```

Comment:
Looks like this does need to be above the code to avoid the failures on CircleCI/Travis?